contrib/x264/module.defs

```diff
@@ -1,8 +1,8 @@
 $(eval $(call import.MODULE.defs,X264,x264))
 $(eval $(call import.CONTRIB.defs,X264))
 
-X264.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs/x264-snapshot-20221003-3100.tar.gz
```

Comment:
This shouldn't be removed, but updated with the current file name, then we'll upload it to the contribs repository.